futures/Cargo.toml

```diff
@@ -20,10 +20,11 @@ appveyor = { repository = "alexcrichton/futures-rs" }
 
 [dependencies]
 futures-core = { path = "../futures-core", version = "0.1.0" }
+futures-channel = { path = "../futures-channel", version = "0.1.0" }
 futures-executor = { path = "../futures-executor", version = "0.1.0" }
 futures-sink = { path = "../futures-sink", version = "0.1.0" }
 futures-util = { path = "../futures-util", version = "0.1.0" }
 
 [features]
-std = []
+std = ["futures-core/std", "futures-executor/std", "futures-sink/std", "futures-util/std"]
```

Comment:
Oh hm this may not actually have much effect unless all the above dependencies are with `default-features = false` as well...

Reply:
ah, you're probably right.